activiti-services/activiti-services-audit/activiti-services-audit-mongo/src/test/java/org/activiti/services/audit/mongo/EventsMongoRepositoryTest.java

```diff
@@ -0,0 +1,224 @@
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.UUID;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.activiti.services.api.events.ProcessEngineEvent;
+import org.activiti.services.audit.mongo.events.ActivityCancelledEventDocument;
+import org.activiti.services.audit.mongo.events.ActivityCompletedEventDocument;
+import org.activiti.services.audit.mongo.events.ActivityStartedEventDocument;
+import org.activiti.services.audit.mongo.events.ProcessCancelledEventDocument;
+import org.activiti.services.audit.mongo.events.ProcessCompletedEventDocument;
+import org.activiti.services.audit.mongo.events.ProcessEngineEventDocument;
+import org.activiti.services.audit.mongo.events.ProcessStartedEventDocument;
+import org.activiti.services.audit.mongo.events.SequenceFlowTakenEventDocument;
+import org.activiti.services.audit.mongo.events.TaskAssignedEventDocument;
+import org.activiti.services.audit.mongo.events.TaskCompletedEventDocument;
+import org.activiti.services.audit.mongo.events.TaskCreatedEventDocument;
+import org.activiti.services.audit.mongo.events.VariableCreatedEventDocument;
+import org.activiti.services.audit.mongo.events.VariableDeletedEventDocument;
+import org.activiti.services.audit.mongo.events.VariableUpdatedEventDocument;
+import org.activiti.services.audit.mongo.repository.EventsRepository;
+import org.activiti.services.core.model.ProcessInstance;
+import org.activiti.services.core.model.ProcessInstance.ProcessInstanceStatus;
+import org.activiti.services.core.model.Task;
+import org.activiti.services.events.ActivityCancelledEventImpl;
+import org.activiti.services.events.ActivityCompletedEventImpl;
+import org.activiti.services.events.ActivityStartedEventImpl;
+import org.activiti.services.events.ProcessCancelledEventImpl;
+import org.activiti.services.events.ProcessCompletedEventImpl;
+import org.activiti.services.events.ProcessStartedEventImpl;
+import org.activiti.services.events.SequenceFlowTakenEventImpl;
+import org.activiti.services.events.TaskAssignedEventImpl;
+import org.activiti.services.events.TaskCompletedEventImpl;
+import org.activiti.services.events.TaskCreatedEventImpl;
+import org.activiti.services.events.VariableCreatedEventImpl;
+import org.activiti.services.events.VariableDeletedEventImpl;
+import org.activiti.services.events.VariableUpdatedEventImpl;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@ContextConfiguration(classes = EventsMongoRepositoryTest.EventsMongoRepositoryConfig.class)
+public class EventsMongoRepositoryTest {
```

Comment:
I think that this test is no longer necessary because you are using the default implementation instead of a custom one. The integration tests present in the starters repository should be enough. I'm going to remove it.